### PR TITLE
feat: poi

### PR DESCRIPTION
**Pull request by EllaMartirosyan**

### CODE REVIEW

File: src/discrete-layer/components/map-container/poi.dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const yupSchema: Record<string, any> = {};
+  Object.keys(poi).forEach(fieldName => {
+    const fieldLabel = `general.dialog-field.${fieldName}.label`;
+    yupSchema[fieldName] = Yup
+    .number()
+    .required(
+      intl.formatMessage(
+        { id: 'validation-general.number' },
+        { fieldName: `<strong>${intl.formatMessage({ id: fieldLabel })}</strong>` }
```
Comment (alebinson) [inline src/discrete-layer/components/map-container/poi.dialog.tsx:47]: extract to function and use it in all <strong> places

**Review by alebinson — APPROVED** [review]
GR8! merge it after the fixes

File: src/discrete-layer/components/map-container/poi.dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    },
+    [onSetOpen]
+  );
+
+  const [poi] = useState<IPOI>({ lon: 0, lat: 0 });
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const yupSchema: Record<string, any> = {};
+  Object.keys(poi).forEach(fieldName => {
+    const fieldLabel = `general.dialog-field.${fieldName}.label`;
```
[the remark this answers is not in the document]
Comment (EllaMartirosyan) [reply src/discrete-layer/components/map-container/poi.dialog.tsx]: Fixed

File: src/discrete-layer/components/map-container/poi.dialog.tsx (hunk 1 above)
Comment (EllaMartirosyan) [reply src/discrete-layer/components/map-container/poi.dialog.tsx]: Done

File: src/common/helpers/formatters.ts [hunk 3]
```diff
@@ -42,3 +42,7 @@ export const dateSerializer: FormatterFunc = (date): string => {
 
   return '-';
 };
+
+export const emphasize: FormatterFunc = (value): string => {
```
Comment (alebinson) [inline src/common/helpers/formatters.ts:46]: emphasizeByHTML

**Review by alebinson — APPROVED** [review]
gr8!
just change the emphasize func() name

File: src/common/helpers/formatters.ts (hunk 3 above)
Comment (EllaMartirosyan) [reply src/common/helpers/formatters.ts]: Fixed

**Pull request merged**